dccrg.hpp

```diff
@@ -8993,6 +8995,18 @@ template <
 								return true;
 							}
 
+							// Instead of splitting paths, check whether our target cell is among the siblings at same transverse indices, i.e. on the same face seen from one level coarser.
+							const uint64_t parent = this->get_parent(cellHere);
+							const std::vector<uint64_t> siblings = this->get_all_children(parent);
+							for(uint64_t sibling : siblings) {
+								if(sibling == cell2) {
+									const Types<3>::indices_t sibling_indices = this->mapping.get_indices(sibling);
+									if(sibling_indices[(dimension+1)%3] == indices2[(dimension+1)%3] && sibling_indices[(dimension+2)%3] == indices2[(dimension+2)%3]) {
```

Comment:
This is bowshock I think, I should compare the indices for [dimension] and not transversely. Testing.